plugins/usbdmx/EurolitePro.cpp

```diff
@@ -176,6 +178,26 @@ bool SynchronousEurolitePro::Init() {
     return false;
   }
 
+  // USB-DMX512-PRO MK2: set baudrate to 250000
+  if (m_is_mk2) {
+    uint16_t divisor = 12;  // = 3000000 / 250000
```

Comment:
I'd be happy to leave this in and let the compiler optimise it out, but I don't think it matters too much either way.

Reply:
Sure, replaced.